app/javascript/src/Policies/components/PoliciesForm.jsx

```diff
@@ -64,8 +64,8 @@ function PoliciesForm ({
       </label>
       <PolicyForm
         className={`PolicyConfiguration-form ${hiddenClass(isNotApicastPolicy(policy))}`}
-        schema={policy.schema}
-        formData={policy.configuration}
+        schema={policy.configuration}
```

Comment:
This is exactly what I mean in the [comment above](https://github.com/3scale/porta/pull/687/files#r263723005), is it a `schema` or a `configuration`? (retorical)

Reply:
Yes, commented as well.